.vscode/launch.json

```diff
@@ -17,14 +17,7 @@
                 "run",
                 "--no-debugger"
             ],
-            "jinja": true
-        },
-        {
-            "name": "Python: Current File",
-            "type": "python",
-            "request": "launch",
-            "program": "${file}",
-            "console": "integratedTerminal"
+            "envFile": "${workspaceFolder}/env/.env",
```

Comment:
Interesting (had to double-check your "running-locally" instructions were correct here). Well done for being thorough!